includes/datastream.inc

```diff
@@ -138,6 +143,7 @@ function islandora_rest_datastream_post_response(array $parameters) {
   $datastream->label = isset($request['label']) ? $request['label'] : $file['name'];
   $datastream->versionable = isset($request['versionable']) ? (bool) $request['versionable'] : TRUE;
   $datastream->mimetype = isset($request['mimeType']) ? $request['mimeType'] : $mime_detect->getMimetype($file['name']);
+  $datastream->checksumType = isset($request['checksumType']) ? $request['checksumType'] : 'DISABLED';
```

Comment:
I think we'll be fine with just the `variable_get()s`; those variables shouldn't exist if checksum isn't installed or is uninstalled later (they're cleared out in `islandora_checksum_uninstall()`), so relying on the default behaviour of `variable_get()` would give us all the functionality we need.

But yeah, I totally forgot about DSIDs to check 😅 

Thanks for this, by the way! I'm surprised this was omitted from the original implementation haha